website/content/en/docs/building-operators/helm/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
 title: Helm
-weight: 3
+weight: 2
```

Comment:
I think the order of the `Building Operators` pages should be:
1. Go
2. Ansible
3. Helm

Reply:
I agree. I was trying to fix just the internal submenu order, good catcher.